conf/logging.properties

```diff
@@ -0,0 +1,12 @@
+java.util.logging.SimpleFormatter.format=%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS [%4$s] %5$s%6$s%n
+.level=ALL
+handlers=java.util.logging.FileHandler
+java.util.logging.FileHandler.level=INFO
+java.util.logging.ConsoleHandler.level=WARNING
+java.util.logging.FileHandler.pattern = /tmp/gigapaxos.log
+java.util.logging.FileHandler.formatter = java.util.logging.SimpleFormatter
+#edu.umass.cs.gigapaxos.PaxosManager.level=FINEST
```

Comment:
Isn't this supposed to be `edu.umass.cs.gigapaxos.PaxosConfig.*` after merging today's pull request in gigapaxos?
Same for `edu.umass.cs.gigapaxos.Reconfigurator.*` as I moved the logger to `ReconfigurationConfig`